qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,12 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def test_session_ls_with_filters(self):
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.mds_asok(['session', 'ls', "%s" % mount_a_client_id])
```

Comment:
`%` operator is deprecated in py 3.0, https://docs.python.org/3.1/whatsnew/2.6.html#pep-3101. Maybe just using `mount_a_client_id` directly is enough i guess?